lib/cdo/data/logging/rake_task_event_logger.rb

```diff
@@ -1,13 +1,14 @@
 require lib_dir 'cdo/data/logging/timed_task_with_logging'
+require 'cdo/aws/metrics'
+
 class RakeTaskEventLogger
   STUDY_TABLE = 'rake_performance'.freeze
-  CURRENT_LOGGING_VERSION = 'v0'.freeze
-
+  CURRENT_LOGGING_VERSION = 'v1'.freeze
   def initialize(rake_task)
     @start_time = 0
     @end_time = 0
     @rake_task = rake_task
-    @enabled = !([:development, :test].include?(rack_env))
+    @enabled = !([:development].include?(rack_env))
```

Comment:
We might be able improve readability to by switching to the `rack_env?()` method. Something like:

``` ruby
@enabled = !rack_env?(:development)
```